views/room.ejs

```diff
@@ -61,6 +61,8 @@
       window.maxUsersPerRoom = <%=maxUsersPerRoom%>;
       window.routedFromStartMeeting = <%=routedFromStartMeeting%>;
       window.userName = "<%=userName%>";
+      window.publishAudio = <%=publishAudio%>;
+      window.publishVideo = <%=publishVideo%>;
       var isMobile = function() { return typeof window.orientation !== 'undefined'; }
```

Comment:
Window.orientation is deprecated 

Refer  [here](https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation)

Reply:
Please if you think its important to remove, create a ticket to address this separately

Reply:
Yeah, I think it's worth logging a separate ticket and addressing this in a separate PR. Good catch.